map/views.py

```diff
@@ -1289,6 +1289,58 @@ def get_tile(self, req, ver, zoom, x, y,
     
         if get_images:
             return rimgs
+
+        if "lslga" in req.GET:
+
+            from PIL import Image, ImageDraw
+            img = Image.open(tilefn)
+
+            ra, dec = wcs.crval
+            pixscale = np.abs(wcs.cd[0]) * 3600
+
+            width, height = img.size
+
+            ralo = ra - ((width / 2) * pixscale / 3600)
+            rahi = ra + ((width / 2) * pixscale / 3600)
+            declo = dec - ((height / 2) * pixscale / 3600)
+            dechi = dec + ((height / 2) * pixscale / 3600)
+
+            import requests
+            json_url = 'http://legacysurvey.org/viewer/lslga/1/cat.json?ralo={}&rahi={}&declo={}&dechi={}'.format(ralo, rahi, declo, dechi)
+            r = requests.get(json_url).json()
```

Comment:
Instead of making a web request (to ourselves), I'll give you a function to call to get the LSLGA galaxies.  Need to slightly refactor the code...

Reply:
Okay, so I finally added the function you need:
```
from map.cats import query_lslga_radecbox
```
https://github.com/legacysurvey/decals-web/blob/master/map/cats.py#L766

Use it like:
```
galaxies = query_lslga_radecbox(ralo, rahi, declo, dechi)
for r in galaxies:
    RA,DEC = r.ra, r.dec
    AB = r.ba
    PA = r.pa
    RAD = r.radius_arcsec
    #....etc...
```

Reply:
Awesome, thanks! A quick question: to incorporate the changes into my branch, would it be preferred for me to merge or rebase?